geomesa-spark/geomesa-spark-converter/src/main/scala/org/locationtech/geomesa/spark/converter/ConverterSpatialRDDProvider.scala

```diff
@@ -84,17 +76,49 @@ class ConverterSpatialRDDProvider extends SpatialRDDProvider with LazyLogging {
     rdd.map(_._2)
   }
 
+  private def computeSftConfig(params: Map[String, String], query: Query) = {
+    if (params.contains(ConverterNameKey)) {
+      // NB: Here we assume that the SFT name and Converter name match.  (And there is no option to rename the SFT.)
+      //  Further, it is assumed that they can loaded from the classpath.
+      val sftName = params.get(ConverterNameKey).orElse(Option(query.getTypeName)).orNull
+      val sft = SimpleFeatureTypeLoader.sftForName(sftName)
+        .getOrElse(throw new IllegalArgumentException(s"Couldn't get look up converter by name for $sftName."))
+      val convertConf = ConverterConfigLoader.confs(sftName).root().render(ConfigRenderOptions.concise())
+
+      (sft, convertConf)
```

Comment:
it would be nice to have this logic somewhere common that we can use it for the tools ingest as well